Storage/Service/EntityStorage.php

```diff
@@ -11,14 +11,11 @@
 
 class EntityStorage implements StorageInterface
 {
-
-    /**@var Registry $doctrine */
-    private $doctrine;
-    /**@var ObjectManager */
+    /** @var ObjectManager */
```

Comment:
this space between /** and @ is important for type hinting. Without the space, the typehint is just a simple comment and not taken into account by PHPStan and PHPStorm